Cover.xs

```diff
@@ -27,23 +27,23 @@ extern "C" {
 #define CALLOP *PL_op
 #endif
 
-#ifndef START_MY_CXT
+#ifndef START_MY_CTX
```

Comment:
unless there's a good reason to use `cxt` over `ctx`, I favor / recommend using `ctx` (both were in the repository, although `cxt` -- here had more instances).